coredb-operator/yaml/crd.yaml

```diff
@@ -47,6 +47,91 @@ spec:
+                        <digit>           ::= 0 | 1 | ... | 9 <digits>          ::= <digit> | <digit><digits> <number>          ::= <digits> | <digits>.<digits> | <digits>. | .<digits> <sign>            ::= "+" | "-" <signedNumber>    ::= <number> | <sign><number> <suffix>          ::= <binarySI> | <decimalExponent> | <decimalSI> <binarySI>        ::= Ki | Mi | Gi | Ti | Pi | Ei
+                          (International System of units; See: http://physics.nist.gov/cuu/Units/binary.html)
+                        <decimalSI>       ::= m | "" | k | M | G | T | P | E
+                          (Note that 1024 = 1Ki but 1000 = 1k; I didn't choose the capitalization.)
+                        <decimalExponent> ::= "e" <signedNumber> | "E" <signedNumber>
+
+                        No matter which of the three exponent forms is used, no quantity may represent a number greater than 2^63-1 in magnitude, nor may it have more than 3 decimal places. Numbers larger or more precise will be capped or rounded up. (E.g.: 0.1m will rounded up to 1m.) This may be extended in the future if we require larger or smaller quantities.
+
+                        When a Quantity is parsed from a string, it will remember the type of suffix it had, and will use the same type again when it is serialized.
+
+                        Before serializing, Quantity will be put in "canonical form". This means that Exponent/suffix will be adjusted up or down (with a corresponding increase or decrease in Mantissa) such that:
+                          a. No precision is lost
+                          b. No fractional digits will be emitted
+                          c. The exponent (or suffix) is as large as possible.
+                        The sign will be omitted unless the number is negative.
+
+                        Examples:
+                          1.5 will be serialized as "1500m"
+                          1.5Gi will be serialized as "1536Mi"
+
+                        Note that the quantity will NEVER be internally represented by a floating point number. That is the whole point of this exercise.
+
+                        Non-canonical values will still parse as long as they are well formed, but will be re-emitted in their canonical form. (So always use canonical form, or don't diff.)
+
+                        This format is intended to make it difficult to use these numbers without writing some sort of special handling code in the hopes that that will cause implementors to also use a fixed point implementation.
+                      type: string
+                    description: 'Limits describes the maximum amount of compute resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/'
+                    type: object
+                  requests:
+                    additionalProperties:
+                      description: |-
```

Comment:
These descriptions can cause unnecessary bloat in our CRD. Go's `controller-gen` has a `maxDescLen` flag that you can set to `0` to disable these descriptions. There doesn't seem to be an equivalent in `kube-derive` at the moment.

It's fine to include them at the moment, but wanted to call this out in case we need to trim things down in the future.